src/renderer/components/editorWithTabs/tabs.vue

```diff
@@ -18,32 +27,131 @@
             <use id="default-close-icon" xlink:href="#icon-close-small"></use>
           </svg>
         </li>
-        <li class="new-file">
-          <svg class="icon" aria-hidden="true"
-            @click.stop="newFile()"
-          >
-            <use xlink:href="#icon-plus"></use>
-          </svg>
-        </li>
       </ul>
     </div>
+    <div
+      class="new-file"
+    >
+      <svg class="icon" aria-hidden="true"
+        @click.stop="newFile()"
+      >
+        <use xlink:href="#icon-plus"></use>
+      </svg>
+    </div>
+  </div>
 </template>
 
 <script>
   import { mapState } from 'vuex'
+  import autoScroll from 'dom-autoscroller'
+  import dragula from 'dragula'
   import { tabsMixins } from '../../mixins'
 
   export default {
+    data () {
+      return {
+        autoScroller: null,
```

Comment:
Because `autoScroller` and `drake` are not used for render the page, so write this way for better performance.

```js
data () {
    this.autoScroller = null
    this.drake = null
    return {}
}
```